src/xrpld/app/tx/applySteps.h

```diff
@@ -242,13 +249,24 @@ struct PreclaimResult
     @return A `PreflightResult` object containing, among
     other things, the `TER` code.
 */
+/** @{ */
```

Comment:
I assume this is an old comment (ditto for line 269)

Reply:
Doxygen grouping.